pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -37,23 +39,81 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
       {!getComponents.isLoading && component && (
         <>
           <div className={styles.heading}>
-            <div>
+            <div className={styles.context}>
               <Heading1>{component.name}</Heading1>
-              <span className={styles.subtitle}>Phasellus tempus. Aenean vulputate eleifend tellus. Sed a libero.</span>
+              <span className={styles.subtitle}>Component organisatie</span>
+              <Paragraph className={styles.description}>{component.description}</Paragraph>
+              <div className={styles.tags}>
+                <Tag tag={component.intendedAudience} />
+                <Tag tag={component.intendedAudience} />
+              </div>
             </div>
-            <div className={styles.tags}>
-              <Tag tag={component.intendedAudience} />
-              <Tag tag={component.developmentStatus} />
+
+            <div className={styles.object}>
```

Comment:
I think `.object` is not declarative enough, _what_ is this div? What does it contain? Maybe it's the `.addToCatalogusContainer`?

Reply:
This div may align to the right of the page (going right to the purple part)

![image](https://user-images.githubusercontent.com/66552932/177322995-865ff125-84a9-45c0-ab0b-a0729392bdce.png)